jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalPostService.java

```diff
@@ -149,11 +150,14 @@ public void deletePost(Post post) {
             branch.clearLastPost();
         }
 
-
         // todo: event API?
         topicDao.saveOrUpdate(topic);
         securityService.deleteFromAcl(post);
-        notificationService.subscribedEntityChanged(topic);
+
+        Set<JCUser> topicSubscribers = new HashSet<>(topic.getSubscribers());
+        topicSubscribers.remove(user);
+        notificationService.subscribedEntityChanged(topic, topicSubscribers);
```

Comment:
This method overloaded. It notifies only those users that are not in the "topicSubscribers" list.
I.e. with the changes being made will be notified only the creator of the post, if he did not remove himself.